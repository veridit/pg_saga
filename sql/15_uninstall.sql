DROP EXTENSION periods;
DROP ROLE periods_unprivileged_user;
