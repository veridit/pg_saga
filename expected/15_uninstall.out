DROP EXTENSION periods;
DROP EXTENSION btree_gist;
DROP ROLE periods_unprivileged_user;
